R/intersectCohorts.R

```diff
@@ -184,6 +184,7 @@ intersectCohorts <- function(cohort,
   cdm <- omopgenerics::dropTable(cdm = cdm, name = tempName)
 
   if (cohortOut |> dplyr::tally() |> dplyr::pull("n") > 0) {
+    class(cohortOut) <- c(class(cohortOut), "cohort_table")
```

Comment:
@KimLopezGuell is it possible to use https://darwin-eu-dev.github.io/omopgenerics/reference/newCohortTable.html here?